boxoffice/models/discount_policy.py

```diff
@@ -43,6 +45,9 @@ class DiscountPolicy(BaseScopedNameMixin, db.Model):
     percentage = db.Column(db.Integer, nullable=True)
     # price-based discount
     is_price_based = db.Column(db.Boolean, default=False, nullable=False)
+    discount_code_base = db.Column(db.Unicode(20), nullable=True)
```

Comment:
This happened to us when we moved from MySQL to PostgreSQL in 2012. Because MySQL's MyISAM storage doesn't support foreign keys, the migrated db in production was missing them for over a year, and because our form validators weren't thorough, we had data corruption until we looked at the schema and noticed it was out of sync (in 2014). We had to dump and re-import data and manually clean up conflicts to fix it.